### PR TITLE
ORM: Improved alloy naming

### PR DESCRIPTION
**Pull request by Henri23729**
## What Does This PR Do

Renames ORM alloys to something more new player friendly.
"Alloy_name+Alloy_name" > "MAT_Name(Alloy_name+Alloy_name)"
## Why It's Good For The Game

"Plasma+Iron" Didn't make it clear for new players (mostly from cargo) that it creates Plasteel, now it does.
## Images of changes

alloys

## Changelog
:cl:
tweak: ORM: Improved alloy naming
/:cl:

### CODE REVIEW

**Review by Sirryan2002 — APPROVED** [review]
code is fine, however, in the future I'd like to see the info in parentheses instead be moved over to using tooltips so there aren't ugly linebreaks in the UI.

**Comment by Henri23729** [comment]
> code is fine, however, in the future I'd like to see the info in parentheses instead be moved over to using tooltips so there aren't ugly linebreaks in the UI.

Oyu is working on adding alloy icons and possibly fixing these linebreaks. He wanted to commit these changes in this PR i think.

File: code/modules/research/designs/smelting_designs.dm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ///////SMELTABLE ALLOYS///////
 
 /datum/design/plasteel_alloy
-	name = "Plasma + Iron alloy"
+	name = "Plasteel(Plasma + Iron)"
```
Comment (SteelSlayer) [inline code/modules/research/designs/smelting_designs.dm:4]:
```suggestion
	name = "Plasteel (Plasma + Iron)"
```

**Review by SteelSlayer — COMMENTED** [review]
Could you add a spaces between all the names and the `(`?

**Review by SteelSlayer — APPROVED** [review]

**Pull request merged**